orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/util/ContextParameterProcessor.java

```diff
@@ -73,6 +73,31 @@ public ContextParameterProcessor(
     return process(source, context, allowUnknownKeys, summary);
   }
 
+  /**
+   * Process pipeline to evaluate spel expressions. Note that 'stages' key is not processed if we
+   * are using spel v4
+   */
+  public Map<String, Object> processPipeline(
+      Map<String, Object> pipeline, Map<String, Object> context, boolean allowUnknownKeys) {
+
+    final String spelEvaluatorKey = "spelEvaluator";
+    ExpressionEvaluationSummary summary = new ExpressionEvaluationSummary();
+    String spelEvaluatorVersion = (String) pipeline.getOrDefault(spelEvaluatorKey, "v3");
```

Comment:
I'd kinda like to see this `"v3"` string pulled into a constant, like `SpelEvaluatorVersion.DEFAULT_VERSION` or something?

Reply:
yup, totally forgot to do that, thanks!